installation_instructions.txt

```diff
@@ -2,4 +2,9 @@ install requirements.txt:
     pip3 install -r requirements.txt
     
 install english language for spacy:
-    python3 -m spacy download en_core_web_sm
\ No newline at end of file
+    python3 -m spacy download en_core_web_sm
+
+install nltk module:
+    python3
+    import nltk
+    nltk.download('all')
```

Comment:
Do we need to download all? Is there an option for english only or something?

Reply:
You can do nltk.download('programName') that is being used

Reply:
I think all were using is 'punkt' and 'stopwords'.
[Source 1](https://stackoverflow.com/questions/37101114/what-to-download-in-order-to-make-nltk-tokenize-word-tokenize-work)
[Source 2](https://www.kite.com/python/answers/how-to-download-nltk-data-in-python)